lethal-squirrels/Bangazon-Site
# Ajproduct

Ticket:#1

## DESCRIPTION
Added ability to add a product.

## STEPS TO TEST
1. Clone ajproduct to your local repo
2. Test functionality
	* if you are not logged into the site, the 'My Products' link should not show up
        *if you are logged in it should show up
        * clicking on the link should take you to the Create Product Page
        * You should be able to input data and when you click Sell, you should be taken to the details 
           page
        * back to list should take you to the list of products
